recipes/libgd/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+import glob
+import os
+from conans import ConanFile, tools, CMake
+
+
+class LibgdConan(ConanFile):
+    name = "libgd"
+    license = "BSD-like"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = ("GD is an open source code library for the dynamic "
+                   "creation of images by programmers.")
+    topics = ("images", "graphics")
+    settings = "os", "compiler", "build_type", "arch"
+    homepage = "https://libgd.github.io"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
```

Comment:
A lot of optional images format are supported, did you try adding those options? https://github.com/libgd/libgd/blob/b079fa06223c3ab862c8f0eea58a968727971988/CMakeLists.txt#L13

Reply:
Datalogics is only using this package for doing GIF I/O. Consequently, we're not doing any kind of testing (even by usage) for the other formats.

The beauty of having a central index of recipes is that interested parties are free to collaborate expand the scope of the recipe if they feel it necessary.